projects/laji/src/app/shared/service/collection.service.ts

```diff
@@ -61,4 +97,35 @@ export class CollectionService extends AbstractCachedHttpService<ICollectionRang
       switchMap(cols => hasMore ? this.fetchWarehouseCollections(page + 1, cols) : of(cols))
     );
   }
+
+  getCollectionsTree(): Observable<any> {
+    return this.graphQlService.query({
+      query: this.TREE_QUERY,
+      errorPolicy: 'all',
+      fetchPolicy: 'cache-first',
+      context: {
+        headers: {
+          'x-timeout': '180000'
+        }
+      }
+
+    }).pipe(
+      map(({data}) => data),
+    );
+  }
+
+  getCollectionsAggregate(page = 1, collections = []): Observable<any> {
```

Comment:
same here

Reply:
Typing is done, as is the fix to to getCollectionsAggreagate.